editor/src/core/workers/ts/ts-worker.spec.ts

```diff
@@ -43,7 +44,7 @@ describe('Typescript worker applies the loaders', () => {
         // Ensure no errors
         for (const builtFile in msg.buildResult) {
           if (msg.buildResult[builtFile].errors.length > 0) {
-            fail(`Build errors found in built file ${builtFile}`)
+            done.fail(`Build errors found in built file ${builtFile}`)
```

Comment:
should this throw an error too?

Reply:
I'm not sure. this test uses `done` so I figured the safest bet is to use `done.fail` so we are not mixing two different ways of using async jest